sensor_msgs/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,129 @@
+This document is a declaration of software quality for the `sensor_msgs` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `sensor_msgs` Quality Declaration
+
+The package `sensor_msgs` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`sensor_msgs` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`sensor_msgs` is not yet at a stable version, i.e. `>= 1.0.0`.
+
+### Public API Declaration [1.iii]
+
+All message and service definition files located in `include`, `msg` and `srv` directories are considered part of the public API.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`sensor_msgs` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`sensor_msgs` contains c++ code and is therefore concerned with ABI stability. It will not break ABI stability within a released ROS distribution.
```

Comment:
```suggestion
`sensor_msgs` contains C++ code and is therefore concerned with ABI stability. It will not break ABI stability within a released ROS distribution.
```

Reply:
Addressed in all